pygfx/renderers/wgpu/shaders/meshshader.py

```diff
@@ -425,7 +425,14 @@ def _encode_morph_texture(self, geometry):
     def get_pipeline_info(self, wobject, shared):
         material = wobject.material
 
-        topology = wgpu.PrimitiveTopology.triangle_list
+        if isinstance(wobject, Line):
```

Comment:
```suggestion
        # The MeshMaterial can be applied to lines and points, so that we can fully support gltf
        if isinstance(wobject, Line):
```